pyproject.toml

```diff
@@ -0,0 +1,82 @@
+[tool.poetry]
+name = "peregrine"
+version = "3.0.1"
+description = "Query interface to get insights into data in Gen3 Commons"
+authors = ["CTDS UChicago <cdis@uchicago.edu>"]
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/uc-cdis/peregrine"
+include = [
+  "NOTICE",
+]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+authutils = "3.1.1"
+datamodelutils = "1.0.0"
```

Comment:
let's use `^` instead of strict pinning wherever possible?

Reply:
Good call 👍